package/tailscale-env

```diff
@@ -2,4 +2,4 @@ PORT="41641"
 TAILSCALED_FLAGS="--tun userspace-networking"
 TAILSCALE_FLAGS=""
 TAILSCALE_AUTOUPDATE="true"
-TAILSCALE_CHANNEL="stable"
\ No newline at end of file
+TAILSCALE_CHANNEL="stable"
```

Comment:
Adding a final newline fixes an issue where  the code [here][1] was yielding a result such as:

```text
PORT="41641"
TAILSCALED_FLAGS="--tun userspace-networking"
TAILSCALE_FLAGS=""
TAILSCALE_AUTOUPDATE="true"
TAILSCALE_CHANNEL="stable"TAILSCALED_FLAGS="--socket=/var/run/tailscale/tailscaled.sock"
```

[1]: https://github.com/SierraSoftworks/tailscale-udm/blob/3de5ce5/install.sh#L54-L57